plugins/source/datadog/go.mod

```diff
@@ -4,8 +4,10 @@ go 1.19
 
 require (
 	github.com/DataDog/datadog-api-client-go/v2 v2.9.0
+	github.com/apache/arrow/go/v13 v13.0.0-20230509040948-de6c3cd2b604
 	github.com/cloudquery/plugin-pb-go v1.0.8
 	github.com/cloudquery/plugin-sdk/v2 v2.7.0
```

Comment:
this shouldn't be there (only as indirect). You need to search where you left v2 reference